openldap-mixin/variables.libsonnet

```diff
@@ -0,0 +1,85 @@
+// variables.libsonnet
+local g = import './g.libsonnet';
+local var = g.dashboard.variable;
+local commonlib = import 'common-lib/common/main.libsonnet';
+local utils = commonlib.utils {
+  labelsToPromQLSelectorAdvanced(labels): std.join(',', ['%s=~"${%s:regex}"' % [label, label] for label in labels]),
+};
+
+{
+  new(this): {
```

Comment:
Depending on the timing, either merge into this pr or a separate small pr afterwards. Just trying to move this PR as the cluster level details are figured out and I can get in to add any additional instance/pod/cluster fields as necessary to improve k8s functionality.

Reply:
OK. I think we should still have some basic level of k8s support in the mixin PR before we hand it off to Grafana even if you're not 100% on the details (I think this is expected by them since this integration will support k8s).

Shouldn't be that hard to add. Here's the PR for PgBouncer which adds k8s support (it's much easier now actually with the new library https://github.com/grafana/jsonnet-libs/pull/1154)